client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -715,10 +717,6 @@ export default {
           Statement: statementFields.join(),
           File: [
             'hash'
-          ].join(),
-          // Segments are only counted, no data needed here.
```

Comment:
Segments are no longer counted?

Reply:
thanks for the advice. done https://github.com/demos-europe/demosplan-core/pull/2549/commits/c4fc0363d3264411df7471d54a2d797781273aa5

Reply:
That was more a review question :-). I won't comment this as this may only lead to confusion why this is explicitly stated

Reply:
oh, I see. the topic is here that the id of segments cannot be a relationship or attribute. it would be in a higher hierarchy I got explained from backend. so it has to be removed here

Reply:
i removed the commit that i put in cause I missunderstood you before https://github.com/demos-europe/demosplan-core/pull/2549/commits/cb0eb76df72eb99a390bf4169b26057299223612